nemo/utils/neural_graph/object_registry.py

```diff
@@ -125,7 +125,7 @@ def __eq__(self, other):
         """
             Checks if two registers have the same content.
 
-            Args:    
```

Comment:
Thanks for the cleanups, interesting, I though that black is taking care of that kind of flaws...